RECOGNITION.md

```diff
@@ -0,0 +1,21 @@
+# Recognizing contributions
+
+The most common form of individual contribution to the Flux project is
+time and effort. Thus, the project recognises people who reported
+issues, fixed bugs, wrote documentation, reviewed pull requests, and
+so on, by thanking them on the Flux website.
+
+However, there are other ways to support the Flux project, especially
+as an organisation, and we would like to be generous in expressing
+gratitude to anyone that helps.
+
+Here are other kinds of contribution that could be recognized:
```

Comment:
This document should include details on the ways we will recognize such contributions. A dedicated section on the website near the individual contributors' avatar gallery is one option. A dedicated page is another one.

Reply:
See https://github.com/fluxcd/community/pull/321#discussion_r1644639112

But again, do we need to specify the exact details of this in this doc?